include/translated-post.php

```diff
@@ -371,6 +371,9 @@ public function create_media_translation( $post_id, $lang ) {
 		$translations[ $lang->slug ] = $tr_id;
 		$this->save_translations( $tr_id, $translations );
 
+		// Loads the strings translations with the attachment's target language.
+		PLL()->load_strings_translations( $lang->slug );
+
```

Comment:
I would move this sooner before we start inserting anything in DB.
And reset to the previous translations before returning.

Reply:
See https://github.com/polylang/polylang/pull/1608/commits/cabf48cf435191af0ae2b876c076ffae3830bee2.